documentation.md

```diff
@@ -0,0 +1,14 @@
+# Documentation
```

Comment:
Can we also add a placeholder, that we will be extending this policy to cover our approach to open-sourcing code as well at some point in the (hopefully, near) future. There is much of our internal code which can be open-sourced with a bit of effort, bringing clear benefits both to others who can re-use it and to us from community feedback. Most of it isn't quite in the right format yet, but the exercise to sanitise and open-source where appropriate is a worthwhile one.

Reply:
Not sure how to best capture intentions. Definitely not this file, maybe the readme, may as GitHub issues?